app.rb

```diff
@@ -0,0 +1,97 @@
+require_relative 'classroom'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+  end
+
+  def list_all_books
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_all_people
+    @people.each do |person|
+      puts "ID: #{person.id}, Name: #{person.name}"
+    end
+  end
```

Comment:
- [ ] According to the wireframe, you need to return these properties as well, kindly use this format:
 `[Student/Teacher] Name: Name, ID: ID, Age: Age`
given:
![image](https://github.com/vanina123/School-libary/assets/91486791/a960e0ec-94f3-4ce7-b599-11246747f07f)
yours:
![image](https://github.com/vanina123/School-libary/assets/91486791/c9b03498-4f63-4d01-aff7-209c08635625)